atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -70,11 +70,12 @@
     int batchSize = AtlasDbConstants.DEFAULT_SWEEP_BATCH_SIZE;
 
     @Option(name = {"--cell-batch-size"},
-            description = "Sweeper cell batch size (default: " + AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE+ ")")
+            description = "Sweeper cell batch size (default: " + AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE + ")")
     int cellBatchSize = AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE;
 
     @Option(name = {"--sleep"},
-            description = "Time to wait in milliseconds after each sweep batch (throttles long-running sweep jobs, default: 0)")
+            description = "Time to wait in milliseconds after each sweep batch "
+                    + "(throttles long-running sweep jobs, default: 0)")
```

Comment:
nit: I think we normally have spaces at the beginning of strings that we concatenate i.e.

```
foo = "bar"
        + " baz";
```